packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -1,17 +1,17 @@
+import React from 'react';
+import { action, autorun, computed, makeObservable, observable, reaction } from 'mobx';
 import {
     config,
     getSavedWorkspaces,
     load,
+    observer as globalObserver,
     removeExistingWorkspace,
     save_types,
     setColors,
-    observer as globalObserver,
 } from '@deriv/bot-skeleton';
 import { isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import { tabs_title, clearInjectionDiv } from 'Constants/load-modal';
-import { action, computed, makeObservable, observable, reaction, autorun } from 'mobx';
-import React from 'react';
+import { clearInjectionDiv, tabs_title } from 'Constants/load-modal';
```

Comment:
this too.. lets revert these changes

Reply:
same as this when i push the import sequence will be edited